### DISCUSSION TITLE
Roadmap to v2.0

### DISCUSSION THREAD

**Discussion by okapies** [pydantic/pydantic#4056] [Feature Request]
The document says some *beta* features (like `default_factory` in [`Field`](https://pydantic-docs.helpmanual.io/usage/models/#field-with-dynamic-default-value)) is not stable, and will be fixed in v2.0. Do you have any plan to release the v2 in the near future?

**Comment by aleksul** [comment]
I believe this may answer your question:
https://slides.com/samuelcolvin/pydantic5

**Comment by samuelcolvin** [comment]
When you say "the document", which document are you referring to?

**Comment by okapies** [comment]
@samuelcolvin I saw it in https://pydantic-docs.helpmanual.io/usage/models/#field-with-dynamic-default-value.

> The `default_factory` argument is in **beta**, it has been added to pydantic in **v1.5** on a **provisional basis**. It may change significantly in future releases and its signature or behaviour will not be concrete until **v2**. Feedback from the community while it's still provisional would be extremely useful; either comment on [#866](https://github.com/samuelcolvin/pydantic/issues/866) or create a new issue.

**Reply by samuelcolvin** [reply]
Default factory will remain. That comment is old and should be removed.
